db/migrations/postgres/000011_create_plugin_key_value_store.down.sql

```diff
@@ -0,0 +1,15 @@
+DO $$BEGIN
+    IF (
+        SELECT column_default::bigint
+        FROM information_schema.columns
+        WHERE table_schema='public'
+        AND table_name='pluginkeyvaluestore'
+        AND column_name='expireat'
+    ) IS NULL THEN
+        ALTER TABLE pluginkeyvaluestore ALTER COLUMN expireat SET DEFAULT 0;
+    END IF;
+END$$;
+
+ALTER TABLE pluginkeyvaluestore DROP COLUMN IF EXISTS expireat;
```

Comment:
Maybe i am missing something, but i am not seeing these alternations anywhere in the deleted code. Is that done from somewhere else?

Reply:
You're totally right, I forgot to remove the `upgrade.go` pieces that modified the table :facepalm: